Formula/blightmud.rb

```diff
@@ -5,8 +5,18 @@ class Blightmud < Formula
   license "GPL-3.0-only"
   depends_on "openssl"
 
-  url "https://github.com/Blightmud/Blightmud/releases/download/v#{version}/blightmud-v#{version}-macos.zip"
-  sha256 "a9bb55128dd61f24bbbee4efca94f1b3dfa45d57f4abda7494a569e824332676"
+  if OS.mac?
+    on_big_sur do
+      url "https://github.com/Blightmud/Blightmud/releases/download/v#{version}/blightmud-v#{version}-macos-bigsur.zip"
```

Comment:
The filename will be `blightmud-v#{version}-macos-11.zip`

Reply:
fixed 👍